lib/nosedrum/command.ex

```diff
@@ -129,5 +129,13 @@ defmodule Nosedrum.Command do
   """
   @callback command(msg :: Message.t(), args :: [String.t()] | any()) :: any()
 
-  @optional_callbacks [parse_args: 1]
+  @doc """
+  An optional callback that returns a list of aliases for the command.
```

Comment:
Can we add a line here explaining that `Nosedrum.Storage` implementations use this in order to install a command at additional paths when `add_command/2` or `remove_command/2` is called?

Having similar documentation in `add_command/2` and `remove_command/2` would be great: I think just a sentence saying that "if a command has the c:Nosedrum.Command.aliases/0 callback defined, they will also be added / removed." would be sufficient.

Reply:
For sure! Just pushed a commit - lmk if those are good enough or you'd like some more detail to them (or resolve if they suite your fancy :slightly_smiling_face: )

Reply:
This is excellent - thank you!